lib/util/fs/index.ts

```diff
@@ -88,14 +88,23 @@ export async function ensureCacheDir(
   envCacheVar?: string
 ): Promise<string> {
   const { cacheDir: adminCacheDir } = getAdminConfig();
-  let envCacheDir = null;
+
+  let cacheDir: string;
   if (envCacheVar) {
     const env = getChildProcessEnv([envCacheVar]);
-    envCacheDir = env[envCacheVar];
+    if (env[envCacheVar]) {
+      cacheDir = env[envCacheVar];
+      logger.trace(`Using cache directory from ${envCacheVar}=${cacheDir}`);
```

Comment:
Better log cachedir as metadata

Reply:
Maybe debug is ok, especially until we have fully transitioned and tested the volumes approach?